Survival box: Replace oxygen bottle and mask with bruise pack (gauze) and matches

du-top opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
<details>
<summary>
	Changes the contents of the survival box; replaces oxygen bottle and mask with gauze and matches.
</summary>
<hr>
In the survival box everyone gets on spawn, there is an oxygen bottle and oxygen mask. (Also a syrette and ERP). Perfect for a space station, but less than useful on a land-based colony with breathable atmosphere (for humans. Let others breath their stupid dirt air, who cares about them anyway. Joking.) Sure, there are fires, but in my eyes, it doesn't make sense to carry everything around for that rather obscure case.
That's why I removed bottle and mask and replaced it with a bruise pack (roll of gauze) and a pack of matches, as a more sensible choice of survival equipment.
I did not touch the extended survival box which is used god-knows-where.
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->
	
<hr>
</details>

## Changelog
:cl:dutop
tweak: In the survival box, switched out oxygen bottle and mask with gauze and matches
<!--add: Added new things
add: Added more things
del: Removed old things
tweak: tweaked a few things
balance: rebalanced something
fix: fixed a few things
soundadd: added a new sound thingy
sounddel: removed an old sound thingy
imageadd: added some icons and images
imagedel: deleted some icons and images
spellcheck: fixed a few typos
code: changed some code
refactor: refactored some code
config: changed some config setting
admin: messed with admin stuff
server: something server ops should know-->
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

du-top commented:
Hmm. Flare sounds nice. But isn't that practically just a very big match? :D Glowstick... neat, too. Damn, now I like every option and I guess all of them would be overkill?

Trilbyspaceclone commented:
I also agree this is a good change

cdb-is-not-good commented:
> Hmm. Flare sounds nice. But isn't that practically just a very big match? :D Glowstick... neat, too. Damn, now I like every option and I guess all of them would be overkill?
> 
> I'd also be open to change the extended survival box that the engineers or medics get. As I play neither, I'd depend on their opinion, though.

It /is/ a giant match yes, but it's also got much better visibility and last a good bit longer. As for the extended kits, i didn't realize adepts/medics got them but honestly they already start with a big cache of supplies at roundstart so im not /too/ sure how useful it'd be to modify the kits beyond what everyone else gets.

du-top commented:
After getting some input here and on discord, I changed some things:
Matches are meh, got it. "but you can make a campfire" isn't really a good point, the MRE is self-heating. After some left-and-right, I changed it to flashlight and a battery. Not because it was k5's idea, but even ChatGPT agreed. Well, that and because a glowstick is lousy, a torch is better, but a flashlight just seems to be the best choice. You can use it for signaling, it is a long, steady light source and not a direct fire hazard.

I _did_ touch the extended survival box now. I kept the extended oxygen tank because I think of the Guild and Medical as the de-facto fire department. But I added the gauze of the flash light, because it would be weird to give them less. Removed the glowstick, though.

(I'll update the PR text next)

AysheDaArtist commented:
Iffy about the light, since it's a free medium cell, and we have lights on the PDA.

The gauze in the box is good for balancing due to how bloodloss works now.

du-top commented:
Small cell. Cheapest and weakest one.

cdb-is-not-good commented:
> Iffy about the light, since it's a free medium cell, and we have lights on the PDA.
> 
> The gauze in the box is good for balancing due to how bloodloss works now.

I'd say it's probably fine, one can get a flashlight in the loadout for fairly cheap and the baseline light small cell is both not that useful for anything more than an emergency and not actually much materials(a tiny bit of silver). I suppose if it becomes a concern down the line I could put together like...I dunno, maybe one of those little handheld lights that you have to wind for charge?

du-top commented:
Well, for once I could remove the additional battery. It was suggested and as no one else weighed in...

The wind-up light was suggested by k5 in discord:
> Ok, idea.
> Flashlight but like. A manual one.
> Gotta spam click it to charge it.
> Uses a tiny bit of nutriment but that comes in the pack as well.

Not sure how it could work smoothly (click can't be switch-on AND winding it, right?) and one would have to sprite a ... well, sprite, so I just thought about it as a nice idea for later.

cdb-is-not-good commented:
It'd be smooth by having an internal battery that starts at 0 and receives charge everytime you wind it. I probably wouldn't make it take nutriment because that's silly, just make each wind take a moment so you can't spam
